src/components/Todos/Todos.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-constant-condition */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[] | null;
+};
+
+export const Todos: React.FC<Props> = ({
+  todos,
+}) => {
+  return (
+    <div>
+      {todos?.map(todo => (
+        <div
+          key={todo.id}
+          id={`${todo.id}`}
+          className={classNames('todo', { completed: todo.completed })}
+        >
+          <label className="todo__status-label">
+            <input
+              type="checkbox"
+              className="todo__status"
+              checked={todo.completed}
+              readOnly
+            />
+          </label>
+          {false ? (
```

Comment:
The 'false' condition should be replaced with a proper condition based on the component's state or props.